### ISSUE TITLE
JDetNet failed while training

### ISSUE THREAD

**Issue by SaddamRaut** [tidsp/caffe-jacinto-models#4]
I am using v0.17

I0810 14:59:47.258464 25579 solver.cpp:352] Iteration 7800 (1.27604 iter/s, 78.3675s/100 iter), 15.1/452.4ep, loss = 5.78276
I0810 14:59:47.258651 25579 solver.cpp:376]     Train net output #0: mbox_loss = 6.02961 (* 1 = 6.02961 loss)
I0810 14:59:47.258673 25579 sgd_solver.cpp:172] Iteration 7800, lr = 0.01, m = 0.9, wd = 0.0001, gs = 1
I0810 15:01:05.190842 25579 solver.cpp:352] Iteration 7900 (1.28319 iter/s, 77.9311s/100 iter), 15.3/452.4ep, loss = 5.22862
I0810 15:01:05.191012 25579 solver.cpp:376]     Train net output #0: mbox_loss = 5.81463 (* 1 = 5.81463 loss)
I0810 15:01:05.191031 25579 sgd_solver.cpp:172] Iteration 7900, lr = 0.01, m = 0.9, wd = 0.0001, gs = 1
I0810 15:02:23.070077 25579 solver.cpp:905] Snapshotting to binary proto file training/voc0712/JDetNet/_ds_PSP_dsFac_32_hdDS8_1/initial/voc0712_ssdJacintoNetV2_iter_8000.caffemodel
I0810 15:02:23.083901 25580 net.cpp:1071] Ignoring source layer mbox_loss
I0810 15:02:23.112282 25579 sgd_solver.cpp:398] Snapshotting solver state to binary proto file training/voc0712/JDetNet/_ds_PSP_dsFac_32_hdDS8_1/initial/voc0712_ssdJacintoNetV2_iter_8000.solverstate
I0810 15:02:23.131805 25579 solver.cpp:635] Iteration 8000, Testing net (#0)
F0810 15:02:23.132831 25579 net.cpp:1081] Check failed: target_blobs[j]->shape() == source_blob->shape() Cannot share param 0 weights from layer 'conv1a/bn'; shape mismatch.  Source param shape is 1 32 1 1 (32); target param shape is 32 (32)
*** Check failure stack trace: ***
    @     0x7f43d58d85cd  google::LogMessage::Fail()
    @     0x7f43d58da433  google::LogMessage::SendToLog()
    @     0x7f43d58d815b  google::LogMessage::Flush()
    @     0x7f43d58dae1e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f43d6879b3e  caffe::Net::ShareTrainedLayersWith()
    @     0x7f43d68edc6c  caffe::Solver::TestDetection()
    @     0x7f43d68f0a87  caffe::Solver::TestAll()
    @     0x7f43d68f1347  caffe::Solver::Step()
    @     0x7f43d68f2d62  caffe::Solver::Solve()
    @     0x7f43d68e3c0a  caffe::P2PSync::InternalThreadEntry()
    @     0x7f43d6472b2c  caffe::InternalThread::entry()
    @     0x7f43d6474ddb  boost::detail::thread_data<>::run()
    @     0x7f43d3dcf5d5  (unknown)
    @     0x7f43d38a06ba  start_thread
    @     0x7f43d40eb41d  clone
    @              (nil)  (unknown)
Aborted (core dumped)

**Comment by mathmanu** [comment]
Please tell me which script are you running.
